data/src/generate_phones_summary.py

```diff
@@ -53,8 +53,10 @@ def main() -> None:
         if file_path.endswith(".md") or file_path.endswith("tsv"):
             continue
         with open(f"{path}/{file_path}", "r", encoding="utf-8") as phone_list:
-            # We subtract one for the final newline.
-            num_of_entries = sum(1 for line in phone_list) - 1
+            # We exclude blank lines and comments.
+            num_of_entries = sum(
+                1 for line in phone_list if line.strip() and line[0] != "#"
```

Comment:
Let me suggest:

```num_of_entries = sum(line.strip().startswith("#") for line in phone_list)```

Reply:
Won't this count the number of comments?

Reply:
oh yes, duh. Try:

```
num_of_entries = sum(not line.strip().startswith("#") for line in phone_list)
```

Note the strategic use of `not` there.

Reply:
Okay, that makes more sense lol. But I think this will (incorrectly) count blank lines, since e.g. `not "\n".lstrip().startswith("#")` is `True`.